lib/index.js

```diff
@@ -155,4 +163,22 @@ exports.register = (server, options) => {
     }
 };
 
-exports.name = NAME;
+exports.addAxiosRoutePreHook = {
+    assign: 'axios',
```

Comment:
Should this prehook assign name 'axios' be configurable?

Reply:
We could. I had considered it, but wasn't sure if we should. I am fine either way.

Reply:
@bnormington Just pushed a refactor to support this concept.